### PR TITLE
chore(j-s): Regression tests

### PR DESCRIPTION
**Pull request by oddsson**
# Regression tests

[Asana](https://app.asana.com/0/1199153462262248/1203046355378011/f)

## What

Add regression tests that do not mock out the backend.

## Why

Today, two team members manually run regression tests. If this can be automated, much time can be saved.

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/web-e2e-regression/src/integration/Court/RestrictionCases/confirmation.spec.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import {
+  RESTRICTION_CASE_CONFIRMATION_ROUTE,
+  CASES_ROUTE,
```
Comment (gudjong) [inline apps/judicial-system/web-e2e-regression/src/integration/Court/RestrictionCases/confirmation.spec.ts:3]: Please remove unused import.

File: apps/judicial-system/web-e2e-regression/src/utils/index.ts [hunk 2]
```diff
@@ -1,6 +1,11 @@
 import { ApolloClient, gql, InMemoryCache } from '@apollo/client'
 
-import { CaseTransition, CaseType } from '@island.is/judicial-system/types'
+import {
+  CaseTransition,
+  CaseType,
+  UpdateCase,
+} from '@island.is/judicial-system/types'
+import { resolve } from 'path'
```
Comment (gudjong) [inline apps/judicial-system/web-e2e-regression/src/utils/index.ts:8]: Consider removing unused import.

**Review by gudjong — CHANGES_REQUESTED** [review]
Minor issues

**Review by Arnarkari93 — APPROVED** [review]
LGTM

**Review by gudjong — APPROVED** [review]

**Pull request merged**